packages/boiler-task-selenium/src/runner/webdriverio/run.js

```diff
@@ -45,22 +45,26 @@ export default function(opts, config, cb) {
 
     // Try to resolve the path or use the external wdio dependency
     try {
-      const webdriverBase = path.dirname(
-        require.resolve('webdriverio')
-      );
+      const webdriverBase = path.dirname(require.resolve('webdriverio'));
 
       // 'webdriverio/build/bin' --> 'webdriver/bin'
       binaryPath = join(webdriverBase, '..', binPath);
     } catch (err) {
       binaryPath = addroot('node_modules/webdriverio', binPath);
     }
 
+    const args = [
+      path.join(__dirname),
+      '--es_staging'
+    ];
+    // Signal to make-config to parallelize tests (and use the dot reporter)
+    if (opt.host === 'hub.browserstack.com') {
+      args.push('--tunnel');
+    }
+
```

Comment:
seems like this `--tunnel` option is what controls if we are using spec of dot reported. We should also be using the `dot` reported locally if `opt.capabilities.length > 1`